[2022-06-13 08:35:58,654] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: a_call_dag.get_sources manual__2022-06-13T08:35:56.101469+00:00 [queued]>
[2022-06-13 08:35:58,669] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: a_call_dag.get_sources manual__2022-06-13T08:35:56.101469+00:00 [queued]>
[2022-06-13 08:35:58,669] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 08:35:58,670] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-13 08:35:58,670] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 08:35:58,706] {taskinstance.py:1377} INFO - Executing <Task(BigQueryToGCSOperator): get_sources> on 2022-06-13 08:35:56.101469+00:00
[2022-06-13 08:35:58,713] {standard_task_runner.py:52} INFO - Started process 13955 to run task
[2022-06-13 08:35:58,718] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'a_call_dag', 'get_sources', 'manual__2022-06-13T08:35:56.101469+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/call_dag.py', '--cfg-path', '/tmp/tmpscfwcsda', '--error-file', '/tmp/tmpirb6_rx9']
[2022-06-13 08:35:58,720] {standard_task_runner.py:80} INFO - Job 24: Subtask get_sources
[2022-06-13 08:35:59,155] {task_command.py:370} INFO - Running <TaskInstance: a_call_dag.get_sources manual__2022-06-13T08:35:56.101469+00:00 [running]> on host 8ef6f3d8e77e
[2022-06-13 08:35:59,886] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=a_call_dag
AIRFLOW_CTX_TASK_ID=get_sources
AIRFLOW_CTX_EXECUTION_DATE=2022-06-13T08:35:56.101469+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-13T08:35:56.101469+00:00
[2022-06-13 08:35:59,904] {bigquery_to_gcs.py:110} INFO - Executing extract of avid-ceiling-348503:wwxt202204.source_list into: ['gs://asia-east2-wwxt202204-97297697-bucket/data/source_list.json']
[2022-06-13 08:35:59,921] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-06-13 08:35:59,935] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-06-13 08:35:59,941] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/google/cloud/hooks/bigquery.py:1921: DeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
  "This method is deprecated. Please use `BigQueryHook.insert_job` method.", DeprecationWarning

[2022-06-13 08:36:00,050] {bigquery.py:1538} INFO - Inserting job ***_1655109360049715_835ade796df5578851260fbc6107e886
[2022-06-13 08:36:02,440] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/bigquery_to_gcs.py", line 128, in execute
    job = hook.get_job(job_id=job_id).to_api_repr()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/common/hooks/base_google.py", line 439, in inner_wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 1470, in get_job
    job = client.get_job(job_id=job_id, project=project_id, location=location)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 2073, in get_job
    timeout=timeout,
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 782, in _call_api
    return call()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/retry.py", line 288, in retry_wrapped_func
    on_error=on_error,
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/retry.py", line 190, in retry_target
    return target()
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/_http/__init__.py", line 494, in api_request
    raise exceptions.from_http_response(response)
google.api_core.exceptions.NotFound: 404 GET https://bigquery.googleapis.com/bigquery/v2/projects/avid-ceiling-348503/jobs/airflow_1655109360049715_835ade796df5578851260fbc6107e886?projection=full&prettyPrint=false: Not found: Job avid-ceiling-348503:airflow_1655109360049715_835ade796df5578851260fbc6107e886
[2022-06-13 08:36:02,521] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=a_call_dag, task_id=get_sources, execution_date=20220613T083556, start_date=20220613T083558, end_date=20220613T083602
[2022-06-13 08:36:02,691] {standard_task_runner.py:97} ERROR - Failed to execute job 24 for task get_sources (404 GET https://bigquery.googleapis.com/bigquery/v2/projects/avid-ceiling-348503/jobs/airflow_1655109360049715_835ade796df5578851260fbc6107e886?projection=full&prettyPrint=false: Not found: Job avid-ceiling-348503:airflow_1655109360049715_835ade796df5578851260fbc6107e886; 13955)
[2022-06-13 08:36:02,789] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-13 08:36:03,076] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
